src/features/search/use-search.ts

```diff
@@ -12,16 +13,37 @@ import { selectSearchTerm } from '@features/search/search-slice';
 import { useDebounce } from '@common/hooks/use-debounce';
 import { isNumeric } from '@common/utils';
 import { Block } from '@stacks/blockchain-api-client';
+import { BTC_BNS_CONTRACT } from '@common/constants';
+import { cvToHex, tupleCV, bufferCVFromString } from '@stacks/transactions';
 
 export const useSearchQuery = (id: string) => {
-  const { searchApi, blocksApi } = useAtomValue(apiClientsState);
+  const { searchApi, blocksApi, nonFungibleTokensApi } = useAtomValue(apiClientsState);
+  const isBtcName = id.endsWith('.btc');
```

Comment:
Not a necessity, but it might be nice to make this case insensitive (and down on lines 34-35)

Reply:
we decided to leave it case-sensitive to match the actual name